engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -765,10 +856,10 @@ public JSONObject getPhysicalOperatorProperties(@PathParam("operatorId") int ope
   @GET
   @Path(PATH_LOGICAL_PLAN)
   @Produces(MediaType.APPLICATION_JSON)
-  public JSONObject getLogicalPlan() throws JSONException, IOException
+  public JSONObject getLogicalPlan(@QueryParam("flatten") String flatten) throws JSONException, IOException
```

Comment:
As discussed all operators are included anyways, so we always "flatten". I think the parameter should be "includeModules" instead.